tutorials/engine/boot_splash.rst

```diff
@@ -0,0 +1,35 @@
+.. _doc_splash_screen_editing:
+
+How to disable the booting splash screen?
+=====
+
+Go to scene>project settings>application>boot_splash
+
+.. image:: /img/boot_splash.png
+
+Uncheck the checkbox or choose a custom splash screen
+
+.. image:: /img/boot_splash_options.png
+
+Adding more splash screen
+-----
+
+Godot is awesome, why not let everyone know about it?
+
+If you want to add one more splash screen after the engine's one, you can create a new scene with the splash screen and set it as the first scene and change to the next scene afterward.
+
+.. image:: /img/boot_splash_main_scene.png
+
+You can do this by creating a scene with a animationplayer node and use the finished() signal for changing the scene, remember to **enable autoplay** in the animationplayer.
```

Comment:
with ***an*** AnimationPlayer node.

... the `finished()` signal ***to change the scene***. sounds better IMO.